kolibri/core/assets/src/views/CoreMenu/CoreMenuOption.vue

```diff
@@ -42,11 +42,26 @@
   export default {
     name: 'CoreMenuOption',
     props: {
-      isDivider: Boolean,
-      label: String,
-      link: String,
-      secondaryText: String,
-      icon: String,
+      isDivider: {
```

Comment:
I think this is pointing out that the `isDivider = true` is a special case and should be separated out as its own component because its inclusion forces us to make `label` and `icon` optional props, when in the more common usage, they are required.